### ISSUE TITLE
v1.2.0 does not work with FASTA files

### ISSUE THREAD

**Issue by kmnip** [bcgsc/ntCard#38]
ntCard version 1.2.0 histograms for FASTA and FASTQ files are different.
I think there is a bug in parsing FASTA files.
Here is an example:

test.fasta
```
>1
CACACACACAAAATCAGTACGTAGCTGATCGTACGATCGTACGATCGTAGCTAGCTAGCTGATGCTAGCTGACTGATCGTAGCTATGTAGCTGATCGATCGTGATCGATCGTACGTAGCTGATGATCGTACGTAGCTAGCTAGCTGATCGATCGATCGTACGTACGTACGTAGTCGATCGTA
```

histogram:
```
F1      0
F0      0
1       9223372036854775808
2       9223372036854775808
3       9223372036854775808
4       9223372036854775808
...
```

test.fastq
```
@1
CACACACACAAAATCAGTACGTAGCTGATCGTACGATCGTACGATCGTAGCTAGCTAGCTGATGCTAGCTGACTGATCGTAGCTATGTAGCTGATCGATCGTGATCGATCGTACGTAGCTGATGATCGTACGTAGCTAGCTAGCTGATCGATCGATCGTACGTACGTACGTAGTCGATCGTA
+1
11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
```

histogram:
```
F1      158
F0      128
1       128
2       0
3       0
4       0
...
```

**Comment by hmohamadi** [comment]
The old version works. 
```
./ntcard -k25 -p test test.fa

F1      158
F0      128
1       128
2       0
3       0
4       0
5       0
6       0
```

**Comment by hmohamadi** [comment]
Found the error in new version. `seq` should be `line`.

**Comment by jowong4** [comment]
@hmohamadi yup, that is correct. I'm suprised the old one didnt fail given that I only changed if the if condition for function.

**Comment by hmohamadi** [comment]
@jowong4 in FASTA processing I designed it to read multiline FASTA records. So we have `line` which can be multiple `seq`.

**Comment by jowong4** [comment]
Fixed in 8ebca4a6e8f42f0fa194d41a66bcee6cf483f341. Thanks @hmohamadi .
